cursive/src/backends/puppet/observed.rs

```diff
@@ -282,7 +282,7 @@ pub trait ObservedPieceInterface {
                     None => s.push(' '),
                     Some(cell) => {
                         if let GraphemePart::Begin(lex) = &cell.letter {
```

Comment:
In addition, I'd rather not use implicit reference pattern here.
```suggestion
                        if let GraphemePart::Begin(ref lex) = cell.letter {
```

Reply:
Good catch! Sorry for not seeing the notification early enough! Thanks for the merge!